src/style.css

```
@@ -1,3 +1,53 @@
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  max-width: 1200px;
+  justify-content: space-between;
+}
```

```suggestion
header {
  display: flex;
  max-width: 1200px;
  justify-content: space-between;
  padding: 0 50px;
}
```
and just add the padding here